packages/ui5-application-writer/src/index.ts

```diff
@@ -78,9 +77,21 @@ async function generate(basePath: string, ui5AppConfig: Ui5App, fs?: Editor): Pr
                 });
             }
         });
+        if (ui5App.appOptions.typescript) {
+            fs.delete(join(basePath, 'webapp/Component.js'));
+            ui5Config.addCustomMiddleware(ui5TsMiddlewares);
+            ui5Config.addCustomTasks(ui5TsTasks);
+            ui5LocalConfig.addCustomMiddleware(ui5TsMiddlewares);
+            ui5LocalConfig.addCustomTasks(ui5TsTasks);
+        }
     }
+
+    // write ui5 yamls
+    fs.write(ui5ConfigPath, ui5Config.toString());
```

Comment:
We already write these files here: https://github.com/SAP/open-ux-tools/pull/626/files#diff-1bff8e3813a4755fbee9efa4f5c4d3cbb9cf7c0b8bbf32e83399a394320e1060R47 
Can we consolidate the writing of these files to one place instead of overwriting?